### PR TITLE
Add Deploy to Azure button

### PR DESCRIPTION
**Pull request by Krusty93**
Requirements before merging:
- public container registry
- this repository made public

### CODE REVIEW

File: azure/deploy-to-azure/app-service.bicep [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+          name: 'ConnectionStrings__ServiceBus'
+          value: serviceBusConnectionString
+        }
+        {
+          name: 'ASPNETCORE_ENVIRONMENT'
+          value: 'Development'
+        }
+        {
+          name: 'DOCKER_REGISTRY_SERVER_URL'
+          value: 'https://index.docker.io'
```
Comment (Krusty93) [inline azure/deploy-to-azure/app-service.bicep:48]: Update when #31 is merged

File: azure/deploy-to-azure/app-service.bicep [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+param appServiceName string
+
+@description('The app service location')
+param location string
+
+@description('Service Bus root connection string')
+param serviceBusConnectionString string
+
+@description('App Service container settings')
+param linuxFxVersion string = 'DOCKER|krusty93/event-bus-explorer:latest'
```
Comment (Krusty93) [inline azure/deploy-to-azure/app-service.bicep:14]: Update when https://github.com/floweSB/event-bus-explorer/pull/31 is merged

Comment (Krusty93) [reply azure/deploy-to-azure/app-service.bicep]: Updated

File: README.md [hunk 3]
```diff
@@ -1 +1,9 @@
-# event-bus-explorer
\ No newline at end of file
+# event-bus-explorer
```
Comment (Krusty93) [inline README.md:1]: rename folder 2 quickstart

**Review by matteopessina — APPROVED** [review]

**Comment by matteopessina** [comment]
@Krusty93 l'ho approvata ma è ancora in draft

**Review by matteopessina — APPROVED** [review]

**Pull request merged**